app/routes/events/components/EventAdministrate/RegistrationTables.js

```diff
@@ -161,20 +224,24 @@ export class RegisteredTable extends Component<Props> {
       },
       {
         title: 'Samtykke',
-        dataIndex: 'photoConsent',
+        dataIndex: 'photoConsents',
         visible: !!event.useConsent,
         center: true,
-        render: (consent) =>
-          consent !== 'UNKNOWN' && (
-            <TooltipIcon
-              content={consent}
-              iconClass={
-                consent === 'PHOTO_CONSENT'
-                  ? cx('fa fa-check', styles.greenIcon)
-                  : cx('fa fa-times', styles.crossIcon)
-              }
-            />
-          ),
+        render: (feedback, registration) => {
+          const eventSemester = getEventSemesterFromStartTime(event.startTime);
+          const photoConsents = registration.user.photoConsents;
+          const LEGACY_photoConsent = registration.LEGACYPhotoConsent;
+
+          return (
+            <div className={styles.consents}>
+              {getConsentIcons(
+                LEGACY_photoConsent,
+                photoConsents,
+                eventSemester
+              )}
```

Comment:
It might just be me, but I _strongly dislike_ this pattern when using react. This can perfectly fine just be a component, which is more readable and react-like (than a function) :smile:.

Reply:
Agree! Fixed. 😄